CHANGELOG.md

```diff
@@ -11,6 +11,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `rust-toolchain.toml` as a single source of truth on toolchain requirements (except Nix builder)
 - Cere Dev Local Testnet config
+- New `set_staking_configs` call in `pallet-ddc-staking` to allow to set DDC Staking bond size by the authority
+
+### Changed
+
+- `pallet-ddc-staking` now requires one fixed size bond for both `Storage` and `Edge` roles instead of the bond limited by the lower boundary only
```

Comment:
I think it's a type here: lower bound*

Reply:
Both `lower bound` and `lower boundary` sound equal to me, do you think it is worth updating it?